scripts/motdgen

```diff
@@ -13,6 +13,15 @@ source /usr/lib/os-release
 mkdir -p /run/coreos
 echo -e "Core\033[38;5;206mO\033[38;5;45mS\033[39m ${GROUP} (${VERSION})" > /run/coreos/motd
 
+mkdir -p /etc/motd.d
```

Comment:
This should be left out or done as a tmpfiles rule, no need to constantly try to create this directory.

https://github.com/coreos/init/tree/master/configs/tmpfiles.d
http://www.freedesktop.org/software/systemd/man/tmpfiles.d.html